### PR TITLE
Bugfixes

### LINKED ISSUE #9751: Alium larvae can't hide
Clicking hide results in you either not going underneath the object, or going underneath of it and immediately popping back out.

### LINKED ISSUE #9757: AI Aliens butcher improperly
``` DM
04:39:01 Runtime detected
bad arg name 'meat' at :
 proc name: gib (/mob/living/simple_animal/hostile/alien/gib)
  usr: Richard Conrad (/mob/living/carbon/human)
  src: the alien drone (/mob/living/simple_animal/hostile/alien/drone)
  usr.loc: the floor (239,261,1) (/turf/simulated/floor)
  src.loc: the floor (240,261,1) (/turf/simulated/floor)
  call stack:
the alien drone (/mob/living/simple_animal/hostile/alien/drone): gib(null)
the alien drone (/mob/living/simple_animal/hostile/alien/drone): Butcher()
the alien drone (/mob/living/simple_animal/hostile/alien/drone): attackby(the kitchen knife (/obj/item/weapon/kitchen/utensil/knife/large), Richard Conrad (/mob/living/carbon/human), "icon-x=16;icon-y=16;left=1;scr...")
Richard Conrad (/mob/living/carbon/human): ClickOn(the alien drone (/mob/living/simple_animal/hostile/alien/drone), "icon-x=16;icon-y=16;left=1;scr...")
the alien drone (/mob/living/simple_animal/hostile/alien/drone): Click(the floor (240,261,1) (/turf/simulated/floor), "mapwindow.map", "icon-x=16;icon-y=16;left=1;scr...")
the alien drone (/mob/living/simple_animal/hostile/alien/drone): Click(the floor (240,261,1) (/turf/simulated/floor), "mapwindow.map", "icon-x=16;icon-y=16;left=1;scr...")

runtime error: bad arg name 'meat'
proc name: gib (/mob/living/simple_animal/hostile/alien/gib)
  usr: Richard Conrad (/mob/living/carbon/human)
  src: the alien drone (/mob/living/simple_animal/hostile/alien/drone)
  usr.loc: the floor (239,261,1) (/turf/simulated/floor)
  src.loc: the floor (240,261,1) (/turf/simulated/floor)
  call stack:
the alien drone (/mob/living/simple_animal/hostile/alien/drone): gib(null)
the alien drone (/mob/living/simple_animal/hostile/alien/drone): Butcher()
the alien drone (/mob/living/simple_animal/hostile/alien/drone): attackby(the kitchen knife (/obj/item/weapon/kitchen/utensil/knife/large), Richard Conrad (/mob/living/carbon/human), "icon-x=16;icon-y=16;left=1;scr...")
Richard Conrad (/mob/living/carbon/human): ClickOn(the alien drone (/mob/living/simple_animal/hostile/alien/drone), "icon-x=16;icon-y=16;left=1;scr...")
the alien drone (/mob/living/simple_animal/hostile/alien/drone): Click(the floor (240,261,1) (/turf/simulated/floor), "mapwindow.map", "icon-x=16;icon-y=16;left=1;scr...")
the alien drone (/mob/living/simple_animal/hostile/alien/drone): Click(the floor (240,261,1) (/turf/simulated/floor), "mapwindow.map", "icon-x=16;icon-y=16;left=1;scr...")
```

### LINKED ISSUE #9851: Dumping toolboxes with clickdrag
(WEB REPORT BY: ziekr REMOTE: 172.93.109.202:7777)

> Revision b5c7f3165a52e8c0f229cf6f7f751042f83f4c27 
> 
> General description of the issue

emptying a toolbox with clickdrag only checks for proximity to the table, not proximity for the toolbox

> Steps to reproduce if possible
> stand near a toolbox and begin clickdragging, step away from the toolbox and next to a table, put mouse over table and release

### LINKED ISSUE #9860: electric chair toggling as ghost
(WEB REPORT BY: ziekr REMOTE: 172.93.109.202:7777)

> Revision 70040064c01e81e7e1e51dd93d2b28a53b3d259e
> 
> General description of the issue

electric chair doesnt prevent ghosts from toggling it on and off

> Steps to reproduce if possible

right click electric chair as ghost
click toggle electric chair

### PR DESCRIPTION
**Pull request by clusterfack**
Fixes #9851
Fixes #9860
Fixes #9757
Fixes #9751 

I'm starting a pass through of all the easy/important bugs to fix

### CODE REVIEW

**Comment by ihadtoregisterforthis** [comment]
Is it **that time** again?

File: code/modules/mob/living/carbon/monkey/monkey.dm [hunk 1]
```diff
@@ -744,3 +744,9 @@
 	if(reagents.has_reagent("methylin"))
 		return 1
 	return 0
+
+/mob/living/carbon/human/reset_layer()
```
Comment (gbasood) [inline code/modules/mob/living/carbon/monkey/monkey.dm:748]: AAAA

File: code/modules/mob/living/carbon/human/human.dm [hunk 2]
```diff
@@ -1797,3 +1797,9 @@
 		return 1
 	else
 		to_chat(src, "<span class='notice'>Something bright flashes in the corner of your vision!</span>")
+
+/mob/living/carbon/human/reset_layer()
```
Comment (gbasood) [inline code/modules/mob/living/carbon/human/human.dm:1801]: AAAAAA

**Comment by clusterfack** [comment]
Did you merge it before Travis or what

**Comment by clusterfack** [comment]
Also if you knew what was wrong why didn't you just fix the damn thing

**Comment by gbasood** [comment]
@clusterfack Because I wasn't thinking
If you check the travis logs, it didn't error OR warn on this, and it's not technically against compiler rules to do this

**Pull request merged**